torch/_higher_order_ops/triton_kernel_wrap.py

```diff
@@ -1342,6 +1344,9 @@ def triton_kernel_wrapper_functional_functionalize(
 triton_kernel_wrapper_functional.fallthrough(DispatchKey.AutogradCUDA)
 triton_kernel_wrapper_functional.fallthrough(DispatchKey.AutogradCPU)
```

Comment:
I wonder when the functional HOP is used over mutation HOP? Should we register to both?